### PR TITLE
feat: framework examples

### LINKED ISSUE #3: Add examples for frameworks
- [x] Svelte Kit
- [x] Next
- [ ] Remix
- [ ] Astro
- [ ] Solid ???

### PR DESCRIPTION
**Pull request by hugomrdias**
closes #3

basically these frameworks do NOT like the nodejs target and they can't handle it, they assume the bundler target always and base64 the wasm blob for SSR.
for client side bundling they can copy the wasm blob to the output because they look for the import statement 
for server side theres no import statement (only fs read file) so they cant figure out where the wasm file is so they cant copy it to the output

### CODE REVIEW

File: packages/wasm/package.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
   "version": "1.0.0",
   "description": "",
   "type": "module",
-  "main": "dist/nodejs/wasm.cjs",
```
Comment (bgins) [inline packages/wasm/package.json:6]: Are we losing anything dropping this? If I recall correctly, just support for projects using older versions of node?

File: examples/sveltekit/src/routes/+layout.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// export const ssr = false
+export const prerender = true
```
Comment (bgins) [inline examples/sveltekit/src/routes/+layout.ts]:
```suggestion
// export const ssr = false
// export const csr = false
export const prerender = true
```

We might want to add the `csr` option as a hint. It also works great!

**Review by bgins — APPROVED** [review]
Looks good and works great! 🎉 

One question, should the commented out examples that use `require` still work? Tried them out quickly and seems like they may not. Thinking of these:

https://github.com/fission-codes/wasm-js/blob/ad7df14675658eab08f019fee95ee3f51941cc93/packages/wasm/test/wasm.test.js#L4-L5

https://github.com/fission-codes/wasm-js/blob/ad7df14675658eab08f019fee95ee3f51941cc93/packages/wasm-test/test/wasm.test.js#L4-L5


We could probably drop support for `require` because folks can `import` nowadays. We may want to mention it in the README if that's what we want to do.

File: packages/wasm/package.json (hunk 1 above)
Comment (hugomrdias) [reply packages/wasm/package.json:6]: yeah and next ignores export maps and uses main for SSR ...

**Pull request merged**